test/utils/isHookCreated.spec.js

```diff
@@ -1,13 +1,15 @@
 import execa from 'execa'
 import fs from 'fs'
+import path from 'path'
 
 import isHookCreated from '../../src/utils/isHookCreated'
 import HOOK from '../../src/commands/hook/hook'
 import * as stubs from './stubs'
 
 describe('isHookCreated', () => {
-  beforeAll(() => {
-    execa.mockReturnValue({ stdout: stubs.gitAbsoluteDir })
+  beforeEach(() => {
+    execa.mockReturnValueOnce({ stdout: stubs.relativeCoreHooksPath })
+    execa.mockReturnValueOnce({ stdout: stubs.gitAbsoluteDir })
```

Comment:
The same as before, we're testing the internal implementation of the util, let's mock the util in this spec!